### ISSUE TITLE
Libcurl not found with Dockerfile-alpine

### ISSUE THREAD

**Issue by pryorda** [freeCodeCamp/devdocs#1029]
When building the Dockerfile-alpine I get this on the build.

```
docker run --rm --name devdocs -p 9292:9292 devdoc
/usr/local/bundle/gems/ffi-1.11.1/lib/ffi/library.rb:145:in `block in ffi_lib': Could not open library 'libcurl': Error loading shared library libcurl: No such file or directory. (LoadError)
Could not open library 'libcurl.so': Error loading shared library libcurl.so: No such file or directory.
Could not open library 'libcurl.so.4': Error loading shared library libcurl.so.4: No such file or directory.
Could not open library 'libcurl.so.4.so': Error loading shared library libcurl.so.4.so: No such file or directory
```

**Comment by pryorda** [comment]
When I fixed that and updated it to install libcurl, I get this error.

``` /devdocs/lib/tasks/sprites.thor:192:in `log_details': undefined method `+' for nil:NilClass (NoMethodError)
	from /devdocs/lib/tasks/sprites.thor:34:in `generate'
	from /usr/local/bundle/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	from /usr/local/bundle/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	from /usr/local/bundle/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	from /usr/local/bundle/gems/thor-0.20.3/lib/thor/invocation.rb:115:in `invoke'
	from /devdocs/lib/app.rb:59:in `block in <class:App>'
	from /usr/local/bundle/gems/sinatra-2.0.5/lib/sinatra/base.rb:1426:in `configure'
	from /devdocs/lib/app.rb:55:in `<class:App>'
	from /devdocs/lib/app.rb:6:in `<top (required)>'
	from /devdocs/config.ru:5:in `require'
	from /devdocs/config.ru:5:in `block in <main>'
	from /usr/local/bundle/gems/rack-2.0.7/lib/rack/builder.rb:55:in `instance_eval'
	from /usr/local/bundle/gems/rack-2.0.7/lib/rack/builder.rb:55:in `initialize'
	from /devdocs/config.ru:in `new'
	from /devdocs/config.ru:in `<main>'
	from /usr/local/bundle/gems/rack-2.0.7/lib/rack/builder.rb:49:in `eval'
	from /usr/local/bundle/gems/rack-2.0.7/lib/rack/builder.rb:49:in `new_from_string'
	from /usr/local/bundle/gems/rack-2.0.7/lib/rack/builder.rb:40:in `parse_file'
	from /usr/local/bundle/gems/rack-2.0.7/lib/rack/server.rb:319:in `build_app_and_options_from_config'
	from /usr/local/bundle/gems/rack-2.0.7/lib/rack/server.rb:219:in `app'
	from /usr/local/bundle/gems/rack-2.0.7/lib/rack/server.rb:354:in `wrapped_app'
	from /usr/local/bundle/gems/rack-2.0.7/lib/rack/server.rb:283:in `start'
	from /usr/local/bundle/gems/rack-2.0.7/lib/rack/server.rb:148:in `start'
	from /usr/local/bundle/gems/rack-2.0.7/bin/rackup:4:in `<top (required)>'
	from /usr/local/bundle/bin/rackup:23:in `load'
	from /usr/local/bundle/bin/rackup:23:in `<main>' ```

**Comment by jmerle** [comment]
Fixed in 5dbb8536069b2956f9c733e627989e6806eb94e3.
